programs/summerofcode/2025.md

```diff
@@ -31,3 +31,17 @@ You can find the project ideas from previous year [here](./2024.md).
 
 ## Ideas
 
+#### ORAS
+
+##### Enhance Java ORAS SDK
+
+- Description: The ORAS project aims to enhance its Java SDK to support a broader range of features from the OCI Distribution spec. This involves implementing missing functionality, improving existing features, and expanding the SDK’s overall capabilities.
+- Expected Outcome:
+  - Implement missing features from the OCI Distribution and Image Specifications, such as [chunked uploads](https://github.com/opencontainers/distribution-spec/blob/main/spec.md#pushing-blobs) and the [Referrers API](https://github.com/opencontainers/distribution-spec/blob/main/spec.md#endpoints)
+  - Improve existing features, robustness and tests to ensure full compatibility with the OCI Distribution and Image Specifications.
+  - Enhance documentation and provide more comprehensive examples.
+  - Add support for additional authentication methods, including using credentials from docker config.json
+- Recommended Skills: java, oci
+- Mentor(s):
```

Comment:
```suggestion
- Recommended Skills: java, oci
- Expected project size: medium (~175 hour projects)
- Mentor(s):
```

Let's drop in a medium size as a placeholder while we figure out what size the project will be.

Reply:
@jonesbusy If the mentorship program requires at least 2 mentors, I can happy to help to become a secondary mentor.

Reply:
Thank you! I will update the PR then